src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.DefinitionTrackingContext.cs

```diff
@@ -43,6 +48,7 @@ public Task SetSearchTitleAsync(string title)
             public Task OnReferenceFoundAsync(SourceReferenceItem reference)
                 => _underlyingContext.OnReferenceFoundAsync(reference);
 
+            [Obsolete("Use ProgressTracker instead", error: false)]
```

Comment:
Do we have a tracking bug tracking cleaning this up? A comment here explaining under what conditions we can delete this would be good.